src/persist/src/indexed/runtime.rs

```diff
@@ -170,82 +223,210 @@ impl RuntimeId {
 }
 
 #[derive(Debug)]
-struct RuntimeHandles {
-    impl_handle: JoinHandle<()>,
-    ticker_handle: tokio::task::JoinHandle<()>,
+enum RuntimeHandles {
+    Full(JoinHandle<()>, tokio::task::JoinHandle<()>),
+    Read(JoinHandle<()>),
```

Comment:
this might also later get a thread/task for dealing with `tail`ing the blob's META and turning it into ListenEvents